### PR TITLE
rena transform is 200x faster

### PR DESCRIPTION
**Pull request by sergulaydore**
Transform is a lot faster with np.bincount instead of np.mean in line 318.

### CODE REVIEW

File: rena.py [hunk 1]
```diff
@@ -194,18 +213,134 @@ def recursive_nearest_agglomeration(data_matrix, connectivity, n_clusters,
...
+        N = X.shape[0]
+        check_is_fitted(self, 'labels_')
+        #unique_labels = np.unique(self.labels_)
+
+#         nX = []
+#         for l in unique_labels:
+#             nX.append(np.mean(X[:, self.labels_ == l], axis=1))
+#         Xred =  np.array(nX).T
+        Xred = np.array([np.bincount(self.labels_, X[i,:])/np.bincount(self.labels_) for i in range(N)])
+
```
Comment (ahoyosid) [inline rena.py]: do you need to calculate np.bincount(self.labels_) again?, It seems that they are already saved in self.sizes_

**Review by ahoyosid — COMMENTED** [review]
There just a minor comment.
Thanks for contributing!

File: rena.py (hunk 1 above)
Comment (sergulaydore) [reply rena.py]: Oh you are right. Fixed it.

**Comment by sergulaydore** [comment]
Good point! Fixed it now.

**Pull request merged**